### PR TITLE
registration-service secrets added

### PR DESCRIPTION
**Pull request by tinakurian**
opening another PR due to branch pairing issues. old PR can be found here: https://github.com/codeready-toolchain/host-operator/pull/263
- - - - - - - - - - - -
This PR adds required configuration for twilio.

This PR is also related to allowing registration-service to have it's configurations set via a configmap and set secrets (particularily for twilio).

Related PR: codeready-toolchain/registration-service#122, PR for toolchain-infra coming soon...

Related Jira: https://issues.redhat.com/browse/CRT-775

### CODE REVIEW

File: pkg/configuration/configuration.go [hunk 1]
```diff
@@ -151,11 +155,32 @@ func LoadConfig(cl client.Client) (*Config, error) {
...
+}
+
+func (c *Config) PrintConfig() {
+	logWithValuesRegServ := log
+	for key, value := range c.GetAllRegistrationServiceParameters() {
+		logWithValuesRegServ = logWithValuesRegServ.WithValues(key, value)
+	}
+
+	logWithValuesRegServ.Info("Please note that these variables can be overridden by the registration-service and " +
+		"hence may differ from the actual values used in the registration-service. Registration Service configuration variables:")
```
[the remark this answers is not in the document]
Comment (tinakurian) [reply pkg/configuration/configuration.go]: sorry forgot to say this was updated here: d35cf3f3426a54446485fecab2bd53ce118260b1

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**